### ISSUE TITLE
The approve bot does not review renovate PRs in Github.com

### ISSUE THREAD

**Issue by samueljseay** [renovatebot/renovate-approve-bot#31]
I've hopefully done all the right things:

1. Added renovate approve bot with the right permissions to the repository
2. I don't have branch restrictions on who can push to the branch
3. The branch has automerge enabled and all statuses succeed

If the approve bot was working, would it immediately add a review or could that be delayed? Thats the only thing I'm not sure about because in the process of testing things some branches have been blocked by statuses or conflicts.

here is a public example branch that I ended up merging myself because no review came from renovate: https://github.com/woocommerce/woocommerce-admin/pull/5045

**Comment by rarkins** [comment]
Sorry, I just found that the bot has stopped approving. Taking a look. In the meantime https://github.com/apps/renovate-approve-2 is still working

**Comment by rarkins** [comment]
Actually now I'm not 100% certain about the 2nd bot after I've been experimenting with them

**Comment by rarkins** [comment]
I think they should both now be working. e.g. I just tested with https://github.com/rarkins/automerge1/pull/1

**Comment by rarkins** [comment]
Note: they're only triggered on PR creation or review dismissal events, so won't be periodically checking for any existing PRs that they "missed" adding reviews to them

**Comment by joshbwlng** [comment]
For what its worth, I've encountered the same issue for both public and private repositories here on GitHub. The second bot seems to be working while the first does not.

**Comment by rarkins** [comment]
If it fails to approve for any PRs *created* from today onwards then please let me know

**Comment by joshbwlng** [comment]
@rarkins I have [this PR](https://github.com/joshbwlng/renovate-test/pull/33) that was opened ~45 minutes ago when only the first bot was installed. It wasn't auto-approved, but a [subsequent PR](https://github.com/joshbwlng/renovate-test/pull/34) opened after installing the second bot was auto-approved by the second bot.

**Comment by rarkins** [comment]
I'm looking into it now. Can you trigger a new PR open?

**Comment by joshbwlng** [comment]
@rarkins 
Here ya go: https://github.com/joshbwlng/renovate-test/pull/35
Looks like both bots approved it this time around, did something change?

**Comment by rarkins** [comment]
I've been fiddling with both bots, including reloading the apps (they run on Glitch), and upgrading/downgrading of probot version. Will leave as-is for now, please alert me again if any problems found.

**Comment by anantakrishna** [comment]
> 1. Added renovate approve bot with the right permissions to the repository

What are the “right permissions”? Can you please suggest?
